src/scripts/main.js

```diff
@@ -22,6 +22,23 @@ const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
   // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    if (data.hasOwnProperty(key)) {
+      const li = document.createElement('li');
+
+      li.textContent = key;
+
+      if (Object.keys(data[key]).length > 0) {
+        createTree(li, data[key]);
+      }
+
+      ul.append(li);
+    }
+  }
+
+  element.append(ul);
 }
```

Comment:
Code Quality Issue: You have a function `createTree` which is recursive. It's good that you've used recursion here as the data structure is a nested object. However, the function is doing too much. It's creating a list, appending list items, and then appending the list to the element. It would be better to split this function into smaller functions, each doing one specific task. This would make your code more readable and maintainable.